django_x509/base/admin.py

```diff
@@ -59,11 +60,25 @@ class CaAdmin(BaseAdmin):
     list_filter = ['key_length', 'digest', 'created']
 
 
-class CertAdmin(BaseAdmin):
+class AbstractCertForm(forms.ModelForm):
+        OPERATION_CHOICES = (
+            ('-----', '-----'),
+            ('import', 'import'),
```

Comment:
change to:
`('import', _('Import existing'))`

You'll have to add the import for `_` (see models if you don't know what that is).